chain-impl-mockchain/src/vote/privacy.rs

```diff
@@ -0,0 +1,13 @@
+use crate::vote::ProofOfCorrectVote;
+use chain_vote::{EncryptedVote, EncryptingVoteKey, Vote};
+use rand_core::{CryptoRng, RngCore};
+
+#[allow(dead_code)]
+pub fn encrypt_vote<R: RngCore + CryptoRng>(
+    rng: &mut R,
+    public_key: &EncryptingVoteKey,
+    vote: Vote,
+) -> (EncryptedVote, ProofOfCorrectVote) {
```

Comment:
May want to `pub use chain_vote::EncryptedVote` in mod `vote` so that all types used here can be imported from mockchain.